pkg/drivers/hyperkit/driver.go

```diff
@@ -240,6 +245,48 @@ func (d *Driver) Start() error {
 	return nil
 }
 
+func (d *Driver) checkForPidFile() error {
+	stateDir := filepath.Join(d.StorePath, "machines", d.MachineName)
+	pidFile := filepath.Join(stateDir, pidFileName)
+
+	if _, err := os.Stat(pidFile); err != nil {
+		log.Infof("clean start, hyperkit pid file doesn't exist: %s", pidFile)
+		return nil
+	}
+
+	log.Warnf("hyperkit pid file exists: %s", pidFile)
+	// we have a pid file, hyperkit might be running!
+	// Make sure the pid written by hyperkit is the same as in the json
+	content, err := ioutil.ReadFile(pidFile)
```

Comment:
Alternatively, since you are just reading one line and casting it to a string, you could also do:

r := bufio.NewReader(file)
line, err := r.ReadString('\n')

Reply:
Why is this better? It doesn't win in number of lines as then I'll need to create & close the file myself, ioutil.ReadFile manages for me :)